train.py

```diff
@@ -114,6 +116,16 @@
         help="Overwrite hyperparameter (e.g. learning_rate:0.01 train_freq:10)",
     )
     parser.add_argument("-uuid", "--uuid", action="store_true", default=False, help="Ensure that the run has a unique ID")
+    parser.add_argument(
+        "--track",
+        type=lambda x: bool(strtobool(x)),
```

Comment:
please use `action="store_true", default=False,` as for the `--no-render` of the enjoy script (unless there is something I'm missing)

Reply:
I would prefer being consistent with the rest of the project, so `--track` if you want to track, otherwise nothing (just like `-optimize`).